recipes/mender-orchestrator/debian-master/control

```diff
@@ -0,0 +1,26 @@
+Source: mender-orchestrator
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends-Indep: cmake, debhelper (>= 10)
+Build-Depends: liblmdb-dev (>= 0.9.7), libc6-dev (>= 2.14), libboost-dev (>= 1.67.0), libboost-filesystem-dev (>= 1.67.0), libboost-log-dev (>= 1.67.0), libboost-regex-dev (>= 1.67.0), libboost-chrono-dev (>= 1.67.0), libboost-thread-dev (>= 1.67.0), libboost-atomic-dev (>= 1.67.0),  libssl-dev (>= 1.1.1), libarchive-dev (>= 3.2.2), libdbus-1-dev
+
+Package: mender-orchestrator-core
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Suggests: mender-client
+Description: Mender Orchestrator Core
+ Mender Orchestrator orchestrates updates across multiple Components in a
+ System, to ensure software consistency across the System as a whole. This
+ package contains the core binary for standalone use cases.
+
+Package: mender-orchestrator
+Architecture: any
+Depends: mender-orchestrator-core
```

Comment:
Same here, I think we need to require the same version. Otherwise `apt-get install mender-orchestrator=1.0` could install `mender-orchestrator-core=2.0` if that's the latest version in the repos.

Reply:
Yes, here is correct. Good catch!